Majasdarbs/GALADARBS/Wheather-app/Kristina-Mezapuke-Gala-darbs/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+  background: #51d0da;
+}
+
+.container-fluid{
+width: 500px;
```

Comment:
maybe better use responsive measurements - vh/vw/% for width. Then it will look better on different screen sizes.